ribs/emitters/operators/_iso_line.py

```diff
@@ -11,30 +11,47 @@ class IsoLineOperator(OperatorBase):
     <https://arxiv.org/abs/1804.03906>`_.
 
     Args:
+        iso_sigma (float): Scale factor for the isotropic distribution used to
+                generate solutions.
+        line_sigma (float): Scale factor for the line distribution used when
+            generating solutions.
         sigma (float or array-like): Standard deviation of the Gaussian
             distribution. Note we assume the Gaussian is diagonal, so if this
             argument is an array, it must be 1D.
         lower_bounds (array-like): Upper bounds of the solution space. Passed in
             by emitter
         upper_bounds (array-like): Upper bounds of the solution space. Passed in
             by emitter
+        parent_type (int): Exposed data member so the selector class knows
+             appropriate parent dimensions.
         seed (int): Value to seed the random number generator. Set to None to
             avoid a fixed seed.
     """
 
-    def __init__(self, iso_sigma, line_sigma, lower_bounds, upper_bounds, seed):
-
+    def __init__(self,
+                 lower_bounds,
+                 upper_bounds,
+                 parent_type=2,
+                 iso_sigma=0.01,
```

Comment:
Can you take out these defaults? It seems a bit dangerous because with these defaults, you can create a GeneticAlgorithmEmitter with `operator="isoline"`, and these defaults would take effect.